src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/InputEmptyBlockDefault.java

```diff
@@ -76,13 +80,13 @@ void method5(int a, int b) {
             case 1:
             case 2:
             case 3:
-            default:       // violation
+            default:        // violation
             {
```

Comment:
Same problem here. Comment is currently above the line where violation actually occurs

Reply:
@bossever I am also facing a similar issue for #9768 where I have to place the `// violation` message **above the line** where **violation** actually occurs because of `line length` getting exceeded. I think right now placing above the **line of violation** might be the way to go. Although we should ask mentors about this.
@timurt Can u pls suggest what can be done in such cases?

Reply:
Resolved.